gui/mainwindow.cpp

```diff
@@ -1142,8 +1142,11 @@ bool MainWindow::getCppcheckSettings(Settings& settings, Suppressions& supprs)
 
         const QString platform = mProjectFile->getPlatform();
         if (platform.endsWith(".xml")) {
-            const QString applicationFilePath = QCoreApplication::applicationFilePath();
-            settings.platform.loadFromFile(applicationFilePath.toStdString().c_str(), platform.toStdString());
+            const std::vector<std::string> paths = {
+                Path::getCurrentPath(), // TODO: do we want to look in CWD?
```

Comment:
Yes.

It's usual that people execute Cppcheck in CI using some script that they can easily run locally also. It makes sense to put related files in the same folder as that script.

Reply:
Will be addressed within #7613.